packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -769,6 +776,7 @@ class _SnackBarState extends State<SnackBar> {
         key: const Key('dismissible'),
         direction: widget.dismissDirection,
         resizeDuration: null,
+        behavior: widget.hitTestBehavior ?? (widget.margin != null ? HitTestBehavior.deferToChild : HitTestBehavior.opaque),
```

Comment:
Nit: I think you should document this default behavior in the docs above for `hitTestBehavior` and/or `margin`.

Reply:
Something like this?